tests/unit/test_textInfos.py

```diff
@@ -0,0 +1,131 @@
+﻿# -*- coding: UTF-8 -*-
+#tests/unit/test_textInfos.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2018 NV Access Limited, Babbage B.V.
+
+"""Unit tests for the textInfos module, its submodules and classes."""
+
+import unittest
+from .textProvider import BasicTextProvider
+import textInfos
+from textInfos.offsets import Offsets
+
+class TestCharacterOffsets(unittest.TestCase):
+	"""
+	Tests for textInfos.offsets.OffsetsTextInfo for its ability to deal with
+	UTF-16 surrogate characters (i.e. whether a surrogate pair is treated as one character).
+	"""
+
+	def test_nonSurrogateForward(self):
+		obj = BasicTextProvider(text="abc")
+		ti = obj.makeTextInfo(Offsets(0, 0)) # Range at a
```

Comment:
I'm concerned that the "Range at a" comment might be misleading, as this is a collapsed range at this point. Maybe the comment could say "Starting at a" or "Range collapsed at a" or similar? Ditto for all the other "Range at" comments below.